content/docs/rfcs/17/README.md

```diff
@@ -203,6 +210,13 @@ With a reasonable approximation of the preceding values, one can set  `max_epoch
 `Network_Delay` and `Clock_Asynchrony` MUST have the same resolution as  `period` .
 By this formulation,  `max_epoch_gap`  indeed measures the maximum number of `epoch`s that can elapse since a message gets routed from its origin to all the other peers in the network.
 
+`acceptable_root_window_size` depends upon the underlying chain's average blocktime, and `Network_Delay`
+
+`acceptable_root_window_size` can be set as $acceptable_root_window_size=block_time/Network_Delay$
+
+By this formulation, `acceptable_root_window_size` will provide an approximation of how many roots can be acceptable by a routing peer.
```

Comment:
I'd be slightly more declarative here

Reply:
Addressed in [3ba4c3c](https://github.com/vacp2p/rfc/pull/539/commits/3ba4c3c75182221f079dab3ec3f935a3464b6803)